### PR TITLE
Update README and add "Now playing" Command

### LINKED ISSUE #579: Add required minimum versions of prerequisites
Could you add a note in the README which minimum or recommended versions of the prerequisites we should use?

We _can_ certainly use the newest version but for compatibility reasons it would be better if you specify it.
One said in the closed Issues that Node.js 17 does have a few problems with muse and 16.x is more recommended.
Your docker file also uses 16.x so why not add it straight to the README?

### LINKED ISSUE #588: Enhancement - Now Playing Command
Added a "Now Playing" Command to show the currently played song.
#587 

This was mainly made for my server but eventually you want it too.
I took the one from the first /play Command and made a new Slash Command out of it.

Text in the screenshot is in German but I've changed it in the code to English:
image

### PR DESCRIPTION
**Pull request by Zagrthos**
Closes #579

Changes werde made to the README File to include the LTS Version of Node.js and a command to run the bot 24/7 in background if you're using a server.

Closes #588

And added a "Now Playing" Slash Command that just shows the Embed from the first /play command to show what currently is played.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,13 +77,16 @@ services:
 
 ### Node.js
 
-**Prerequisites**: Node.js, ffmpeg
+**Prerequisites**:
+* Node.js 16.x (16.x is recommended because of LTS. 17.x seems to make problems)
```
Comment (codetheweb) [inline README.md:81]:
```suggestion
* Node.js (16.x is recommended because it's the current LTS version)
```

File: README.md [hunk 2]
```diff
@@ -77,13 +77,16 @@ services:
...
+* ffmpeg
 
 1. `git clone https://github.com/codetheweb/muse.git && cd muse`
 2. Copy `.env.example` to `.env` and populate with values
 3. I recommend checking out a tagged release with `git checkout v[latest release]`
 4. `yarn install` (or `npm i`)
 5. `yarn start` (or `npm run start`)
+	* `(npm run start&)` to run the bot 24/7 on a server
```
Comment (codetheweb) [inline README.md:89]:
```suggestion

```

There's a lot of ways to run this on a server besides just as a background process, like [PM2](https://pm2.keymetrics.io/). I'd rather not recommend anything specific here.

**Review by codetheweb — CHANGES_REQUESTED** [review]
Please keep different changes in separate PRs in the future.

I left a comment in https://github.com/codetheweb/muse/issues/588.

**Comment by Zagrthos** [comment]
Sorry! Initially I wanted to keep the changed separate but GitHub decided to merge them in one PR. Guess I create extra branches in the future for single changes to avoid this.

**Review by codetheweb — APPROVED** [review]
I made a few tweaks, if it looks good to you I'll go ahead and merge. :)

**Comment by Zagrthos** [comment]
Already approved it! :)

**Pull request merged**